build.sbt

```diff
@@ -39,11 +33,19 @@ lazy val cross = crossProject(JSPlatform, JVMPlatform).in(file(".")).
     scalacOptions ++= Seq("-feature", "-deprecation", "-language:implicitConversions", "-language:higherKinds"),
     scalacOptions ++= {
       scalaVersion.value match {
-        case v if v.startsWith("2.13") => Nil
-        case _ => Seq("-Ypartial-unification")
+        case v if v.startsWith("2.12") => Seq("-Ypartial-unification")
+        case v if v.startsWith("3") => Seq("-source", "3.0-migration")
```

Comment:
This is to make Dotty shut up about missing parens in lambdas. I didn't fix them to keep the PR as small as possible.